pkg/controller/util/status.go

```diff
@@ -27,6 +27,16 @@ var (
 	PhaseClusterExpanding = "Expanding Capacity"
 	// PhaseDeleting is used when cluster is deleting
 	PhaseDeleting = "Deleting"
+	// PhaseConnecting is used when cluster is connecting to external cluster
+	PhaseConnecting = "Connecting"
+	//PhaseConnected is used when cluster has connected to external cluster
+	PhaseConnected = "Connected"
+)
+
+const (
```

Comment:
Exported Constants should have comments. 
Check lint errors.

Reply:
Added the comments